ItsJonQ/g2
# Types: create-styles and styles

This PR adds types to create-styles and styles.

Proof it works 😎 

<img width="730" alt="Captura de Tela 2020-11-13 às 15 31 17" src="https://user-images.githubusercontent.com/24264157/99130487-53399600-25c5-11eb-88e7-3092c3b5d2cb.png">
<img width="499" alt="Captura de Tela 2020-11-13 às 15 31 02" src="https://user-images.githubusercontent.com/24264157/99130493-55035980-25c5-11eb-8659-1a9e51c35cd1.png">

These screenshots are taken from inside the `styles` package.

### Testing instructions

* Manually check generated types to make sure they look right (I mean skim them probably).
* Consume the types from `styles` and pay special attention to `styled` and `get` which are the most complicated types we generate. Make sure they continue to work as expected with the appropriate type hints 😎 
* Make sure the components still work 😱
